core/24.0.1/docker-entrypoint.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+set -e
+
+if [[ "$1" == "bitcoin-cli" || "$1" == "bitcoin-tx" || "$1" == "bitcoind" || "$1" == "test_bitcoin" ]]; then
+	mkdir -p "$BITCOIN_DATA"
+
+	CONFIG_PREFIX=""
+	if [[ "${BITCOIN_NETWORK}" == "regtest" ]]; then
+		CONFIG_PREFIX=$'regtest=1\n[regtest]'
+	elif [[ "${BITCOIN_NETWORK}" == "testnet" ]]; then
+		CONFIG_PREFIX=$'testnet=1\n[test]'
+	elif [[ "${BITCOIN_NETWORK}" == "mainnet" ]]; then
```

Comment:
Starting bitcoind with `-mainnet=1` errors with `Invalid parameter -mainnet=1`. Was this supposed to be `signet`?

Reply:
Specifying it in the config file like this, it gives a warning:
`Ignoring unknown configuration value mainnet`

Reply:
@andrewtoth thanks for your advice! You are totally right. I added the option for signet.

For mainnet I still add `[main]` to the `bitcoin.conf` (but don't add `mainnet=1` anymore).